lib/rules/color-contrast-matches.js

```diff
@@ -63,7 +63,7 @@ if (node.getAttribute('id')) {
 	}
 }
 
-if (axe.commons.text.visible(virtualNode, false, true) === '') {
+if (axe.commons.text.visibleVirtual(virtualNode, false, true) === '') {
```

Comment:
Shouldn't `visible` call `visibleVirtual`? It would help backwards compatibility.